.github/CODEOWNERS

```diff
@@ -1 +1,4 @@
-* @AlexandrePicosson @AurelienGasser @cboillet-dev @Kelvin-M @mblottiere @samlesu @sergebouchut2 @ThibaultFy @oleobal @Milouu @guilhem-barthes
+# historic code owners
+* @AlexandrePicosson @cboillet-dev @Kelvin-M @mblottiere @samlesu @sergebouchut2
```

Comment:
Can you remove me also, in order to avoiding to flood my notifications?
Feel free to reach me / mention me, when you need though!

Reply:
Maybe we reached a point where the historic owners are not necessary? I'm genuinely asking because I did not significantly contribute to any review in the last months. And as Serge wrote above, we are all reachable through other mean if necessary.

Reply:
Yes, that is my opinion too. We'll discuss this on squad day next monday, I expect y'all be removed from codeowners at that point.